setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+"""
+Standard build script.
+"""
+
+from __future__ import print_function
+
+from os.path import dirname, join
+
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    print("You must have setuptools installed to use setup.py. Exiting...")
+    raise SystemExit(1)
+
+__docformat__ = 'restructuredtext'
+
+# Import requirements
+with open(join(dirname(__file__), 'requirements.txt')) as f:
+    required = f.read().splitlines()
+
+setup(
+    name="python-owasp-zap-v2.4",
```

Comment:
Really, IMHO, the better choice si to use a standard name and unify the version. So there's a "version" param in setup(..). Currently in Pypi are available 3 different libraries:
<img width="1009" alt="shell" src="https://cloud.githubusercontent.com/assets/4076390/14737237/5ec83998-087b-11e6-801c-b3438b2b7a74.png">

This is not a the correct way to send an update to Pypi. I'll change the name to:
`setup(name="python-owasp-zap"
            ...`
And only update the "version" param:
`setup(...
           version=1.0.0
           ...`

For Python developers is more intuitive and easy if only one library is available, with different versions release. 

When we update for a new version, not change the name name of library, only de version param, doing:
`python setup.py sdist upload`

Of, for the first release (if we change the name):
`python setup.py sdist register upload`